### ISSUE TITLE
Problem when $httpProvider.defaults.withCredentials is enabled

### ISSUE THREAD

**Issue by azzlack** [khaines/angular-applicationinsights#18]
When you set `$httpProvider.defaults.withCredentials = true` every call to the AI service will fail with the following message:

```
XMLHttpRequest cannot load https://dc.services.visualstudio.com/v2/track. 
A wildcard '*' cannot be used in the 'Access-Control-Allow-Origin' header when the credentials flag is true. 
Origin 'https://localhost:44322' is therefore not allowed access.
```

Maybe it is possible to disable that flag when posting to AI?

**Comment by khaines** [comment]
Yes, this should be investigated. Thank you for reporting it.

**Comment by BringerOD** [comment]
I am having the same issue.  Has anyone found a work around?

**Comment by khaines** [comment]
Finally has some time to open this project up to write and test the fix. I should have the patch out in the next hour.

**Comment by khaines** [comment]
fixed in v0.2.5
